cactusbot/services/beam/handler.py

```diff
@@ -92,8 +95,9 @@ def __init__(self, channel):
             return
```

Comment:
`event` will never be `None` - `str.split()` always returns a `tuple` - no exceptions. No need to have it here.